packages/codelabs/src/intermediate/lit-html.md

```diff
@@ -74,79 +74,106 @@ You should already know how to create a web component using `LitElement`. Go ahe
+Create your URL with the itemId above, and visit it from the browser. You should see the
+API return a JSON response, which looks similar to this:
+
+```json
+{
+  "by": "[username]",
+  "id": 22115410,
+  "kids": ["..."],
+  "parent": 22106482,
+  "text": "lorem ipsum",
+  "time": 1579678389,
+  "type": "comment"
+}
+```
 
-Create your URL, and visit it from the browser. You should see the API return a JSON response. If you're stuck, you can try the [getting started](https://newsapi.org/docs/get-started) page for the news API. Don't follow any of the code examples yet, we will look into this in the next step
+<aside class="notice">
+  Get the full <a href="https://github.com/HackerNews/API">HackerNews API Documentation</a>.
+  There is additional detailed information about the endpoints and a list of all available
+  endpoints. These endpoints are not required for this tutorial.
+</aside>
 
-## Fetching news articles
+## Fetching HackerNews comments
 
-Besides displaying UI, web components can also use any of the available javascript APIs. We are going to use the `fetch` function to make an HTTP request to retrieve our news articles. `fetch` is an asynchronous operation, and it costs system resources. We, therefore, need to be a bit more careful about when and how we use it.
+Besides displaying UI, web components can also use any of the available javascript APIs. We are going to use the `fetch` function to make an HTTP request to retrieve our HackerNews comments. `fetch` is an asynchronous operation, and it costs system resources. We, therefore, need to be a bit more careful about when and how we use it.
 
 `LitElement` has several lifecycle methods available for this, some of them will be familiar to you by now. See [this page](/faq/lit-element-lifecycle.html) for a full overview and reference of all the available lifecycle methods.
 
 We could trigger our `fetch` in the constructor since it's run only once. But because it's a best practice to not perform any side effects there, it's better to use the `connectedCallback.` Because this method can be called multiple times during an element's lifecycle, we should be careful to trigger a `fetch` only when the data hasn't already been fetched before.
 
 ### Tasks to complete this step:
 
-- Fetch articles from the news API
-- Render the fetched articles as JSON
+- Fetch five comments from the news API
+- Render the fetched comments as JSON
+
+<aside class="notice">
+  You can use the item id <i style="font-weight: 700;">22115410</i> as a starting point.
+</aside>
 
 ### Tips
 
+The HN API doesn't distinguish its items, thus it might be difficult to know if a
+received object is indeed a comment. To verify that the data fetched from the HackerNews
+API is a comment you can simply use an if-conditional and check against its data.type.
+
+```js
+if(data.type !== "comment") return false;
```

Comment:
Can we add more code context around this code snippet? Where do people add/use this code?

Reply:
Yes i can add more context. Stay tuned